newsroom/wire/views.py

```diff
@@ -87,6 +88,10 @@ def get_view_data():
     company_id = str(user["company"]) if user and user.get("company") else None
     user_folders = get_user_folders(user, "wire") if user else []
     company_folders = get_company_folders(company, "wire") if company else []
+    products = get_products_by_company(company, product_type="wire") if company else []
+
+    if user:
```

Comment:
As this method is only used in views decorated with `@login_required`, we could assume that a user should always be available here, otherwise it might be an auth issue.

Do you see any scenario where a user would not be defined here (and in agenda `get_view_data`)?